c8/c8ql.py

```diff
@@ -337,3 +337,20 @@ def response_handler(resp):
             return True
 
         return self._execute(request, response_handler)
+
+    def export_data_query(self, query, bind_vars):
```

Comment:
Hello Edgar see if you can remove the changes that have come into this branch from other branches.. While creating a branch I think you are doing some small mistake since it carries all changes from your other branches

Reply:
Once that is done, do let me know I would like to do some tests on the insert functions